### PR TITLE
BACK-1669 compute operation depends on batch concurrency configuration

### PR DESCRIPTION
**Pull request by mbasire-ledger**
## What is this about?

Compute operation lasts much longer than other step of Interpreter.compute.

The changes proposed does not enhance the performance of initial sync of a 49k+ transactions accounts on some infra.

  * Operations saved by batches of 100 to 1000 operations.
  * The size depends on the batch concurrency configuration
  * all saving operations moved to OperationService
  * log duration of compute steps
  * the operation computation still lasts long in some environment

## Cute picture of lama
lama

### CODE REVIEW

File: coins/bitcoin/interpreter/src/main/scala/co/ledger/lama/bitcoin/interpreter/Interpreter.scala [hunk 1]
```diff
@@ -98,16 +99,43 @@ class Interpreter(
...
+
+      _ <- log.info(s"[$accountId] Flagging inputs and outputs belong")
+
+      _ <- flaggingService.flagInputsAndOutputs(accountId, addresses)
+
+      flaggingEnd <- clock.monotonic(TimeUnit.MILLISECONDS)
+
+      _ <- operationService.deleteUnconfirmedOperations(accountId)
+
+      cleanUnconfirmedEnd <- clock.monotonic(TimeUnit.MILLISECONDS)
```
Comment (estrauser-ledger) [inline coins/bitcoin/interpreter/src/main/scala/co/ledger/lama/bitcoin/interpreter/Interpreter.scala:112]: Tu veux garder les clock ?

**Comment by estrauser-ledger** [comment]
Tu voulais pas retirer les foreign keys ?

File: coins/bitcoin/interpreter/src/main/scala/co/ledger/lama/bitcoin/interpreter/Interpreter.scala (hunk 1 above)
Comment (mbasire-ledger) [reply coins/bitcoin/interpreter/src/main/scala/co/ledger/lama/bitcoin/interpreter/Interpreter.scala:112]: Why not ? It is not costly and it logs useful metrics on a single line.

**Comment by mbasire-ledger** [comment]
> Tu voulais pas retirer les foreign keys ?

Yes. But after a cross check with Eric, it seems the runtime infrastucture may reduce the time far better that the `remove foreign key` trick. On his machine, the step `compute operation` duration is not so huge that on my machine.

**Review by nukky94 — APPROVED** [review]

**Pull request merged**